### ISSUE TITLE
Database query error when trying to edit page

### ISSUE THREAD

**Issue by alex-mashin** [WikiApiary/WikiApiary#234]
A database query error has occurred. This may indicate a bug in the software.

```
Query:
SELECT 1 FROM `page` WHERE (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) LIMIT 1
Function: ApiStashEdit::checkCache
Error: 1054 Unknown column 'Array' in 'where clause' (db-local)
```

while saving an edit (via Edit Source) to "Traditio" page (https://wikiapiary.com/wiki/Traditio).

**Comment by alex-mashin** [comment]
The same thing happens if I try to edit with form:
`A database error has occurred. Did you forget to run maintenance/update.php after upgrading? See: https://www.mediawiki.org/wiki/Manual:Upgrading#Run_the_update_script Query: SELECT 1 FROM `page` WHERE (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) LIMIT 1 Function: ApiStashEdit::checkCache Error: 1054 Unknown column 'Array' in 'where clause' (db-local)`
Perhaps, you really should run maintenance/update.php?

**Comment by jthingelstad** [comment]
I saw this issue myself on another page, but I can’t easily replicate it. I was just able to edit the page you are trying. (Also the bots have been editing thousands of pages).

https://wikiapiary.com/w/index.php?title=Traditio&type=revision&diff=2144301&oldid=1610430

Definitely ran `update.php`, but just for fun I’m going to run it again.

```
Updating for wikiapiary.com...
MediaWiki 1.25.1 Updater

Your composer.lock file is up to date with current dependencies!
Going to run database updates for mw_wikiapiary
Depending on the size of your database this may take a while!
...echo_subscription doesn't exist.
Turning off Content Handler DB fields for this part of upgrade.
...have ipb_id field in ipblocks table.
...have ipb_expiry field in ipblocks table.
...already have interwiki table
...indexes seem up to 20031107 standards.
...have rc_type field in recentchanges table.
...index new_name_timestamp already set on recentchanges table.
...have user_real_name field in user table.
...querycache table already exists.
...objectcache table already exists.
...categorylinks table already exists.
...have pagelinks; skipping old links table updates
...il_from OK
...have rc_ip field in recentchanges table.
...index PRIMARY already set on image table.
...have rc_id field in recentchanges table.
...have rc_patrolled field in recentchanges table.
...logging table already exists.
...have user_token field in user table.
...have wl_notificationtimestamp field in watchlist table.
...watchlist talk page rows already present.
...user table does not contain user_emailauthenticationtimestamp field.
...page table already exists.
...have log_params field in logging table.
...logging table has correct log_title encoding.
...have ar_rev_id field in archive table.
...have page_len field in page table.
...revision table does not contain inverse_timestamp field.
...have rev_text_id field in revision table.
...have rev_deleted field in revision table.
...have img_width field in image table.
...have img_metadata field in image table.
...have user_email_token field in user table.
...have ar_text_id field in archive table.
...page_namespace is already a full int (int(11)).
...ar_namespace is already a full int (int(11)).
...rc_namespace is already a full int (int(11)).
...wl_namespace is already a full int (int(11)).
...qc_namespace is already a full int (int(11)).
...log_namespace is already a full int (int(11)).
...have img_media_type field in image table.
...already have pagelinks table.
...image table does not contain img_type field.
...already have unique user_name index.
...user_groups table exists and is in current format.
...have ss_total_pages field in site_stats table.
...user_newtalk table already exists.
...transcache table already exists.
...have iw_trans field in interwiki table.
...wl_notificationtimestamp is already nullable.
...index times already set on logging table.
...have ipb_range_start field in ipblocks table.
...no page_random rows needed to be set
...have user_registration field in user table.
...templatelinks table already exists
...externallinks table already exists.
...job table already exists.
...have ss_images field in site_stats table.
...langlinks table already exists.
...querycache_info table already exists.
...filearchive table already exists.
...have ipb_anon_only field in ipblocks table.
...index rc_ns_usertext already set on recentchanges table.
...index rc_user_text already set on recentchanges table.
...have user_newpass_time field in user table.
...redirect table already exists.
...querycachetwo table already exists.
...have ipb_enable_autoblock field in ipblocks table.
...index pl_namespace on table pagelinks includes field pl_from.
...index tl_namespace on table templatelinks includes field tl_from.
...index il_to on table imagelinks includes field il_from.
...have rc_old_len field in recentchanges table.
...have user_editcount field in user table.
...page_restrictions table already exists.
...have log_id field in logging table.
...have rev_parent_id field in revision table.
...have pr_id field in page_restrictions table.
...have rev_len field in revision table.
...have rc_deleted field in recentchanges table.
...have log_deleted field in logging table.
...have ar_deleted field in archive table.
...have ipb_deleted field in ipblocks table.
...have fa_deleted field in filearchive table.
...have ar_len field in archive table.
...have ipb_block_email field in ipblocks table.
...index cl_sortkey on table categorylinks includes field cl_from.
...have oi_metadata field in oldimage table.
...index usertext_timestamp already set on archive table.
...index img_usertext_timestamp already set on image table.
...index oi_usertext_timestamp already set on oldimage table.
...have ar_page_id field in archive table.
...have img_sha1 field in image table.
...protected_titles table already exists.
...have ipb_by_text field in ipblocks table.
...page_props table already exists.
...updatelog table already exists.
...category table already exists.
...category table already populated.
...have ar_parent_id field in archive table.
...have user_last_timestamp field in user_newtalk table.
...protected_titles table has correct pt_title encoding.
...profiling table has pf_memory field.
...have ss_active_users field in site_stats table.
...ss_active_users user count set...
...have ipb_allow_usertalk field in ipblocks table.
...pl_namespace, tl_namespace, il_to indices are already UNIQUE.
...change_tag table already exists.
...tag_summary table already exists.
...valid_tag table already exists.
...user_properties table already exists.
...log_search table already exists.
...have log_user_text field in logging table.
...l10n_cache table already exists.
...index ls_field_val already set on log_search table.
...index change_tag_rc_tag already set on change_tag table.
...have rd_interwiki field in redirect table.
...transcache tc_time already converted.
...*_mime_minor fields are already long enough.
...iwlinks table already exists.
...index iwl_prefix_title_from already set on iwlinks table.
...have ul_value field in updatelog table.
...have iw_api field in interwiki table.
...iwl_prefix key doesn't exist.
...have cl_collation field in categorylinks table.
...categorylinks up-to-date.
...collations up-to-date.
...msg_resource table already exists.
...module_deps table already exists.
...ar_page_revid key doesn't exist.
...index ar_revid already set on archive table.
...ll_lang is up-to-date.
...user_last_timestamp is already nullable.
...index user_email already set on user table.
...up_property in table user_properties already modified by patch patch-up_property.sql.
...uploadstash table already exists.
...user_former_groups table already exists.
...index type_action already set on logging table.
...have rev_sha1 field in revision table.
...batch conversion of user_options: nothing to migrate. done.
...user table does not contain user_options field.
...have ar_sha1 field in archive table.
...index page_redirect_namespace_len already set on page table.
...have us_chunk_inx field in uploadstash table.
...have job_timestamp field in job table.
...index page_user_timestamp already set on revision table.
...have ipb_parent_block_id field in ipblocks table.
...index ipb_parent_block_id already set on ipblocks table.
...category table does not contain cat_hidden field.
...have rev_content_format field in revision table.
...have rev_content_model field in revision table.
...have ar_content_format field in archive table.
...have ar_content_model field in archive table.
...have page_content_model field in page table.
Content Handler DB fields should be usable now.
...site_stats table does not contain ss_admins field.
...recentchanges table does not contain rc_moved_to_title field.
...sites table already exists.
...have fa_sha1 field in filearchive table.
...have job_token field in job table.
...have job_attempts field in job table.
...have us_props field in uploadstash table.
...ug_group in table user_groups already modified by patch patch-ug_group-length-increase-255.sql.
...ufg_group in table user_former_groups already modified by patch patch-ufg_group-length-increase-255.sql.
...index pp_propname_page already set on page_props table.
...index img_media_mime already set on image table.
...iwl_prefix_title_from index is already non-UNIQUE.
...index iwl_prefix_from_title already set on iwlinks table.
...have ar_id field in archive table.
...have el_id field in externallinks table.
...have rc_source field in recentchanges table.
...index log_user_text_type_time already set on logging table.
...index log_user_text_time already set on logging table.
...have page_links_updated field in page table.
...have user_password_expires field in user table.
...have pp_sortkey field in page_props table.
...recentchanges table does not contain rc_cur_time field.
...index wl_user_notificationtimestamp already set on watchlist table.
...have page_lang field in page table.
...have pl_from_namespace field in pagelinks table.
...have tl_from_namespace field in templatelinks table.
...have il_from_namespace field in imagelinks table.
...img_major_mime in table image already modified by patch patch-img_major_mime-chemical.sql.
...oi_major_mime in table oldimage already modified by patch patch-oi_major_mime-chemical.sql.
...fa_major_mime in table filearchive already modified by patch patch-fa_major_mime-chemical.sql.
...user_id is already unsigned int.
...rc_comment in table recentchanges already modified by patch patch-editsummary-length.sql.
...maps_layers table already exists.
...math table already exists.
...mathoid table already exists.
...approved_revs table already exists.
...echo_event table already exists.
...echo_email_batch table already exists.
...echo_target_page table already exists.
...event_agent field does not exist in echo_event table, skipping modify field patch.
...event_variant in table echo_event already modified by patch /srv/www/mediawiki/public_html/w/extensions/Echo/db_patches/patch-event_variant_nullability.sql.
...event_extra in table echo_event already modified by patch /srv/www/mediawiki/public_html/w/extensions/Echo/db_patches/patch-event_extra-size.sql.
...event_agent_ip in table echo_event already modified by patch /srv/www/mediawiki/public_html/w/extensions/Echo/db_patches/patch-event_agent_ip-size.sql.
...have etp_id field in echo_target_page table.
...have notification_bundle_base field in echo_notification table.
...echo_event table does not contain event_timestamp field.
...have eeb_event_hash field in echo_email_batch table.
...have event_page_id field in echo_event table.
...index echo_event_type already set on echo_event table.
...index echo_user_timestamp already set on echo_notification table.
...translate_sections table already exists.
...have trs_order field in translate_sections table.
...revtag table already exists.
...translate_groupstats table already exists.
...index trs_page_order already set on translate_sections table.
...trs_page key doesn't exist.
...translate_reviews table already exists.
...translate_groupreviews table already exists.
...translate_tms table already exists.
...translate_metadata table already exists.
...translate_messageindex table already exists.
...index tgs_lang already set on translate_groupstats table.
...have tgs_proofread field in translate_groupstats table.
...translate_stash table already exists.
Setting up standard database configuration for SMW ...

Selected storage engine is "SMWSQLStore3" (or an extension thereof)

Checking table `smw_object_ids` ...
   Table already exists, checking structure ...
   ... field smw_id is fine.
   ... field smw_namespace is fine.
   ... field smw_title is fine.
   ... field smw_iw is fine.
   ... field smw_subobject is fine.
   ... field smw_sortkey is fine.
   ... field smw_proptable_hash is fine.
   ... done.
Checking table `smw_concept_cache` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_id is fine.
   ... done.
Checking table `smw_prop_stats` ...
   Table already exists, checking structure ...
   ... field p_id is fine.
   ... field usage_count is fine.
   ... done.
Checking table `smw_di_number` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field p_id is fine.
   ... field o_serialized is fine.
   ... field o_sortkey is fine.
   ... done.
Checking index structures for table `smw_di_number` ...
   ... index s_id,p_id is fine.
   ... index p_id,o_sortkey is fine.
   ... done.
Checking table `smw_di_blob` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field p_id is fine.
   ... field o_blob is fine.
   ... field o_hash is fine.
   ... done.
Checking index structures for table `smw_di_blob` ...
   ... index s_id,p_id is fine.
   ... index p_id,o_hash is fine.
   ... done.
Checking table `smw_di_bool` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field p_id is fine.
   ... field o_value is fine.
   ... done.
Checking index structures for table `smw_di_bool` ...
   ... index s_id,p_id is fine.
   ... index p_id,o_value is fine.
   ... done.
Checking table `smw_di_uri` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field p_id is fine.
   ... field o_serialized is fine.
   ... done.
Checking index structures for table `smw_di_uri` ...
   ... index s_id,p_id is fine.
   ... index p_id,o_serialized is fine.
   ... done.
Checking table `smw_di_time` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field p_id is fine.
   ... field o_serialized is fine.
   ... field o_sortkey is fine.
   ... done.
Checking index structures for table `smw_di_time` ...
   ... index s_id,p_id is fine.
   ... index p_id,o_sortkey is fine.
   ... done.
Checking table `smw_di_coords` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field p_id is fine.
   ... field o_serialized is fine.
   ... field o_lat is fine.
   ... field o_lon is fine.
   ... done.
Checking index structures for table `smw_di_coords` ...
   ... index s_id,p_id is fine.
   ... index p_id,o_serialized is fine.
   ... index o_lat,o_lon is fine.
   ... done.
Checking table `smw_di_wikipage` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field p_id is fine.
   ... field o_id is fine.
   ... done.
Checking index structures for table `smw_di_wikipage` ...
   ... index s_id,p_id is fine.
   ... index p_id,o_id is fine.
   ... index o_id is fine.
   ... done.
Checking table `smw_fpt_type` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_serialized is fine.
   ... done.
Checking index structures for table `smw_fpt_type` ...
   ... index s_id is fine.
   ... index o_serialized is fine.
   ... done.
Checking table `smw_fpt_unit` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_blob is fine.
   ... field o_hash is fine.
   ... done.
Checking index structures for table `smw_fpt_unit` ...
   ... index s_id is fine.
   ... index o_hash is fine.
   ... done.
Checking table `smw_fpt_conv` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_blob is fine.
   ... field o_hash is fine.
   ... done.
Checking index structures for table `smw_fpt_conv` ...
   ... index s_id is fine.
   ... index o_hash is fine.
   ... done.
Checking table `smw_fpt_pval` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_blob is fine.
   ... field o_hash is fine.
   ... done.
Checking index structures for table `smw_fpt_pval` ...
   ... index s_id is fine.
   ... index o_hash is fine.
   ... done.
Checking table `smw_fpt_list` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_blob is fine.
   ... field o_hash is fine.
   ... done.
Checking index structures for table `smw_fpt_list` ...
   ... index s_id is fine.
   ... index o_hash is fine.
   ... done.
Checking table `smw_fpt_serv` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_blob is fine.
   ... field o_hash is fine.
   ... done.
Checking index structures for table `smw_fpt_serv` ...
   ... index s_id is fine.
   ... index o_hash is fine.
   ... done.
Checking table `smw_fpt_ask` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_id is fine.
   ... done.
Checking index structures for table `smw_fpt_ask` ...
   ... index s_id is fine.
   ... index o_id is fine.
   ... done.
Checking table `smw_fpt_askde` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_serialized is fine.
   ... field o_sortkey is fine.
   ... done.
Checking index structures for table `smw_fpt_askde` ...
   ... index s_id is fine.
   ... index o_sortkey is fine.
   ... done.
Checking table `smw_fpt_asksi` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_serialized is fine.
   ... field o_sortkey is fine.
   ... done.
Checking index structures for table `smw_fpt_asksi` ...
   ... index s_id is fine.
   ... index o_sortkey is fine.
   ... done.
Checking table `smw_fpt_askfo` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_blob is fine.
   ... field o_hash is fine.
   ... done.
Checking index structures for table `smw_fpt_askfo` ...
   ... index s_id is fine.
   ... index o_hash is fine.
   ... done.
Checking table `smw_fpt_askst` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_blob is fine.
   ... field o_hash is fine.
   ... done.
Checking index structures for table `smw_fpt_askst` ...
   ... index s_id is fine.
   ... index o_hash is fine.
   ... done.
Checking table `smw_fpt_askdu` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_serialized is fine.
   ... field o_sortkey is fine.
   ... done.
Checking index structures for table `smw_fpt_askdu` ...
   ... index s_id is fine.
   ... index o_sortkey is fine.
   ... done.
Checking table `smw_fpt_subp` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_id is fine.
   ... done.
Checking index structures for table `smw_fpt_subp` ...
   ... index s_id is fine.
   ... index o_id is fine.
   ... done.
Checking table `smw_fpt_subc` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_id is fine.
   ... done.
Checking index structures for table `smw_fpt_subc` ...
   ... index s_id is fine.
   ... index o_id is fine.
   ... done.
Checking table `smw_fpt_inst` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_id is fine.
   ... done.
Checking index structures for table `smw_fpt_inst` ...
   ... index s_id is fine.
   ... index o_id is fine.
   ... done.
Checking table `smw_fpt_redi` ...
   Table already exists, checking structure ...
   ... field s_title is fine.
   ... field s_namespace is fine.
   ... field o_id is fine.
   ... done.
Checking index structures for table `smw_fpt_redi` ...
   ... index s_title,s_namespace is fine.
   ... index o_id is fine.
   ... done.
Checking table `smw_fpt_sobj` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_id is fine.
   ... done.
Checking index structures for table `smw_fpt_sobj` ...
   ... index s_id is fine.
   ... index o_id is fine.
   ... done.
Checking table `smw_fpt_impo` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_blob is fine.
   ... field o_hash is fine.
   ... done.
Checking index structures for table `smw_fpt_impo` ...
   ... index s_id is fine.
   ... index o_hash is fine.
   ... done.
Checking table `smw_fpt_uri` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_serialized is fine.
   ... done.
Checking index structures for table `smw_fpt_uri` ...
   ... index s_id is fine.
   ... index o_serialized is fine.
   ... done.
Checking table `smw_fpt_conc` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field concept_txt is fine.
   ... field concept_docu is fine.
   ... field concept_features is fine.
   ... field concept_size is fine.
   ... field concept_depth is fine.
   ... field cache_date is fine.
   ... field cache_count is fine.
   ... done.
Checking index structures for table `smw_fpt_conc` ...
   ... index s_id is fine.
   ... done.
Checking table `smw_fpt_mdat` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_serialized is fine.
   ... field o_sortkey is fine.
   ... done.
Checking index structures for table `smw_fpt_mdat` ...
   ... index s_id is fine.
   ... index o_sortkey is fine.
   ... done.
Checking table `smw_fpt_cdat` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_serialized is fine.
   ... field o_sortkey is fine.
   ... done.
Checking index structures for table `smw_fpt_cdat` ...
   ... index s_id is fine.
   ... index o_sortkey is fine.
   ... done.
Checking table `smw_fpt_ledt` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_id is fine.
   ... done.
Checking index structures for table `smw_fpt_ledt` ...
   ... index s_id is fine.
   ... index o_id is fine.
   ... done.
Checking table `smw_fpt_sci_cite_key` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_blob is fine.
   ... field o_hash is fine.
   ... done.
Checking index structures for table `smw_fpt_sci_cite_key` ...
   ... index s_id is fine.
   ... index o_hash is fine.
   ... done.
Checking table `smw_fpt_sci_cite_reference` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_blob is fine.
   ... field o_hash is fine.
   ... done.
Checking index structures for table `smw_fpt_sci_cite_reference` ...
   ... index s_id is fine.
   ... index o_hash is fine.
   ... done.
Checking table `smw_fpt_sci_cite_text` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_blob is fine.
   ... field o_hash is fine.
   ... done.
Checking index structures for table `smw_fpt_sci_cite_text` ...
   ... index s_id is fine.
   ... index o_hash is fine.
   ... done.
Checking table `smw_fpt_sci_cite` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_id is fine.
   ... done.
Checking index structures for table `smw_fpt_sci_cite` ...
   ... index s_id is fine.
   ... index o_id is fine.
   ... done.
Checking table `smw_fpt_sci_doi` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_blob is fine.
   ... field o_hash is fine.
   ... done.
Checking index structures for table `smw_fpt_sci_doi` ...
   ... index s_id is fine.
   ... index o_hash is fine.
   ... done.
Checking table `smw_fpt_sci_pmcid` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_blob is fine.
   ... field o_hash is fine.
   ... done.
Checking index structures for table `smw_fpt_sci_pmcid` ...
   ... index s_id is fine.
   ... index o_hash is fine.
   ... done.
Checking table `smw_fpt_sci_pmid` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_blob is fine.
   ... field o_hash is fine.
   ... done.
Checking index structures for table `smw_fpt_sci_pmid` ...
   ... index s_id is fine.
   ... index o_hash is fine.
   ... done.
Checking table `smw_fpt_sci_oclc` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_blob is fine.
   ... field o_hash is fine.
   ... done.
Checking index structures for table `smw_fpt_sci_oclc` ...
   ... index s_id is fine.
   ... index o_hash is fine.
   ... done.
Checking table `smw_fpt_sci_viaf` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_blob is fine.
   ... field o_hash is fine.
   ... done.
Checking index structures for table `smw_fpt_sci_viaf` ...
   ... index s_id is fine.
   ... index o_hash is fine.
   ... done.
Checking table `smw_fpt_sci_olid` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_blob is fine.
   ... field o_hash is fine.
   ... done.
Checking index structures for table `smw_fpt_sci_olid` ...
   ... index s_id is fine.
   ... index o_hash is fine.
   ... done.
Checking table `smw_ftp_sesp_pageid` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_serialized is fine.
   ... field o_sortkey is fine.
   ... done.
Checking index structures for table `smw_ftp_sesp_pageid` ...
   ... index s_id is fine.
   ... index o_sortkey is fine.
   ... done.
Checking table `smw_ftp_sesp_euser` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_id is fine.
   ... done.
Checking index structures for table `smw_ftp_sesp_euser` ...
   ... index s_id is fine.
   ... index o_id is fine.
   ... done.
Checking table `smw_ftp_sesp_cuser` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_id is fine.
   ... done.
Checking index structures for table `smw_ftp_sesp_cuser` ...
   ... index s_id is fine.
   ... index o_id is fine.
   ... done.
Checking table `smw_ftp_sesp_subp` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_id is fine.
   ... done.
Checking index structures for table `smw_ftp_sesp_subp` ...
   ... index s_id is fine.
   ... index o_id is fine.
   ... done.
Checking table `smw_ftp_sesp_nrev` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_serialized is fine.
   ... field o_sortkey is fine.
   ... done.
Checking index structures for table `smw_ftp_sesp_nrev` ...
   ... index s_id is fine.
   ... index o_sortkey is fine.
   ... done.
Checking table `smw_ftp_sesp_userreg` ...
   Table already exists, checking structure ...
   ... field s_id is fine.
   ... field o_serialized is fine.
   ... field o_sortkey is fine.
   ... done.
Checking index structures for table `smw_ftp_sesp_userreg` ...
   ... index s_id is fine.
   ... index o_sortkey is fine.
   ... done.
Database initialized successfully.

Setting up internal property indices ...
   ... space for internal properties already allocated.
   ... writing entries for internal properties ... done.
Internal properties initialized successfully.
...site_stats is populated...done.
...Update 'populate rev_len and ar_len' already logged as completed.
...Update 'populate rev_sha1' already logged as completed.
...img_sha1 column of image table already populated.
...protocol-relative URLs in externallinks table already fixed.
...fa_sha1 column of filearchive table already populated.
...*_from_namespace column of backlink tables already populated.
Failed to set the local repo temp zone container to be private.
Purging caches...done.

Done in 0:02.
```

If it still doesn't work, can you try logging out and back in? Let me know what happens.

**Comment by alex-mashin** [comment]
It didn't help.

**Comment by jthingelstad** [comment]
My suspicion is going to be ConfirmEdit. I’m guessing it has a bug. Will look deeper later today. @kghbln if you have any ideas let me know.

**Comment by alex-mashin** [comment]
Before the issue is fixed, could you please, since you are able to edit the page, change the site domain (including its API) from traditio-ru.org to traditio.wiki?

**Comment by jthingelstad** [comment]
@alex-mashin [done](https://wikiapiary.com/w/index.php?title=Traditio&type=revision&diff=2144390&oldid=2144301).

**Comment by alex-mashin** [comment]
Thanks.

**Comment by mwjames** [comment]
Have a look at https://phabricator.wikimedia.org/T105597.

**Comment by kghbln** [comment]
Oops, I did not get the note for this conversation. I actually opened a bug report for this here https://github.com/WikiApiary/wiki/issues/4

**Comment by revi** [comment]
Got the same thing few minutes ago while putting `{{delete|duplicate}}` to [[위키쨩 2]].

```
A database query error has occurred. This may indicate a bug in the software.

Query:
SELECT 1 FROM `page` WHERE (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) OR (Array) LIMIT 1
Function: ApiStashEdit::checkCache
Error: 1054 Unknown column 'Array' in 'where clause' (db-local)

```

**Comment by kghbln** [comment]
@thingles Updating WikiApiary to latest MW should fix this

**Comment by jthingelstad** [comment]
Should be fixed now with upgrade to 1.25.3.
